### PR TITLE
[#472] Fix OCP 3.11 mirror download URL

### PR DESCRIPTION
**Pull request by mcarlett**
Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- [x] Code is self-descriptive and/or documented

### CODE REVIEW

File: core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java [hunk 1]
```diff
@@ -47,10 +47,19 @@ private String determineClientUrl(final ClusterVersionInfo versionInfo) {
...
-        if (versionInfo.getOpenshiftVersion().startsWith("3")) {
+        final String openshiftVersion = versionInfo.getOpenshiftVersion();
+        if (openshiftVersion.startsWith("3")) {
             // OpenShift 3
-            return String.format("%s/%s/%s/oc.tar.gz", OCP3_CLIENTS_URL, versionInfo.getOpenshiftVersion(),
-                    getSystemTypeForOCP3());
+            final String systemTypeForOCP3 = getSystemTypeForOCP3();
+            String downloadUrl = String.format("%s/%s/%s/oc.tar.gz", OCP3_CLIENTS_URL, openshiftVersion,
+                    systemTypeForOCP3);
+            // if the generated download URL is not working (404 or 403 response code) try to concatenate -1 to the version
```
Comment (mnovak1) [inline core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java:56]: It's just weird Openshift client version does have such a suffix in the mirror. Do we have any guidence from Openshift team for this?

**Review by simkam — APPROVED** [review]
Thanks for bringing it back, I accidentally omitted it in #465. Sorry about it.

File: core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java (hunk 1 above)
Comment (simkam) [reply core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java:56]: I'm only guessing but it might be a build number.

Comment (mcarlett) [reply core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java:56]: I just found the version as **"OpenShift Web Console"** version in the `https://[host]:8443/console/about` page

Comment (mnovak1) [reply core/src/main/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolver.java:56]: Ok, let's merge it. I think it's more temporary solution. We'll see.

**Pull request merged**